app/templates/show-project.hbs

```diff
@@ -72,40 +72,35 @@
         <div class="grid-x grid-padding-x">
           <div class="cell medium-6">
             <h2>Actions</h2>
-            <ul class="no-bullet">
+            <ul class="actions-list no-bullet">
               {{#each model.actions as |action|}}
-                <li class="grid-x grid-padding-small">
+                <li class="grid-x grid-padding-small action">
                   <div class="cell shrink">
                     <span class="action-icon fa-layers fa-fw">
                       {{fa-icon 'circle' fixedWidth=true class='dark-gray'}}
                       {{fa-icon 'file-alt' fixedWidth=true class='light-gray' transform='shrink-7'}}
                     </span>
                   </div>
                   <div class="cell auto">
-                    <p class="no-margin">
-                      <strong>
-                        <a href="#">
-                          {{action.dcp_name}}
-                        </a>
-                      </strong>
-                      <sup class="dark-gray">
-                        {{icon-tooltip tip=(lookup-action-type action.actioncode)}}
-                      </sup>
-                    </p>
-                    <p class="text-small dark-gray">
-                      {{if action.dcp_prefix action.dcp_prefix "No Prefix"}}
-                      <span class="medium-gray">|</span>
-                      {{#if action.dcp_ulurpnumber}}
-                        {{#if (eq action.statuscode "Approved")}}
-                          <a href={{build-url "cpcReport" action.dcp_ulurpnumber}} target="_blank">{{action.dcp_ulurpnumber}}</a><sup class="dark-gray">{{icon-tooltip tip='ULURP Number (click on this number to access the City Planning Commision Report).'}}</sup>
+                    <h5 class="no-margin">{{action.dcp_name}}<sup class="dark-gray">{{icon-tooltip tip=(lookup-action-type action.actioncode)}}</sup></h5>
+                    <p class="text-tiny label-group">
+                      <span class="label light-gray">
+                        {{if action.dcp_prefix action.dcp_prefix "No Prefix"}}
+                      </span>{{!-- remove whitespace
+                  --}}<span class="label light-gray">
```

Comment:
Growing concerned abt code styling here. We have no linters installed for handlebars, but we should start remembering too. My bad!

Reply:
Yeah, I wasn't sure about how to best remove that whitespace. Handlebars whitespace control (`~`) doesn't help us here because of the hardcoded span. If that conditional were block level, it could control the whitespace, but the inline simplicity would be lost.